src/documents/index.py

```diff
@@ -225,15 +225,19 @@ def _get_query_filter(self):
 
         user_criterias = [query.Term("has_owner", False)]
         if "user" in self.query_params:
-            user_criterias.append(query.Term("owner_id", self.query_params["user"]))
-            user_criterias.append(
-                query.Term("viewer_id", str(self.query_params["user"])),
-            )
+            if self.query_params["is_superuser"]:  # superusers see all docs
+                user_criterias = []
+            else:
+                user_criterias.append(query.Term("owner_id", self.query_params["user"]))
+                user_criterias.append(
+                    query.Term("viewer_id", str(self.query_params["user"])),
+                )
         if len(criterias) > 0:
```

Comment:
The coverage gap is because `criterias` is empty.  I think a test would need one of the "special" strings set to hit this line

Reply:
Ah thanks, ok

Reply:
Whoops, wrong kinda change. I'll fix later!

Reply:
Wait, im not crazy, shouldn't that fix it now? `criterias` won't be empty with `&document_type__id__none=1`

Reply:
Edited mere seconds after you commented

Reply:
Ha it said edited 5 minutes ago so I was confused. 🤪